oslabs-beta/janusQL
# User login and Registration

## **Types of Changes**



## **Purpose**
1. Add user account creation endpoints
2. Add user login routing to landing page

## **Approach**
User Login:

Altered state declaration and added event listener on React side to submit button that posts to DB. Upon validation, re-routes user to landing page.

Added account validation middleware and endpoints to server

## **How to test**
Commands:
(in separate terminals)
npm run dev -> to start server
npm run dev:react -> to start react
npm run dev:electron -> to boot up electron


Enter valid credentials, click Login.

Expected Behavior:
Reroute user to landing page.
If invalid credentials entered, display error.
